guides/common/modules/proc_enabling-the-template-sync-plugin.adoc

```diff
@@ -2,12 +2,30 @@
 = Enabling the Template Sync plugin
 
 .Procedure
-. To enable the plugin on your {ProjectServer}, enter the following command:
+ifdef::satellite[]
+* Enable the plugin on your {ProjectServer}:
+endif::[]
+ifndef::satellite[]
+. Enable the plugin on your {ProjectServer}:
+endif::[]
```

Comment:
```suggestion
. Enable the plugin on your {ProjectServer}:
```
The whole module is excluded from satellite builds.

Reply:
Thx, applied.